controllers/party_controller.py

```diff
@@ -66,13 +66,13 @@ def add_survey(party_id, enrolment_code):
 
 
 def get_party_by_email(email):
-    logger.debug('Retrieving party by email address', email=email)
+    logger.debug('Retrieving party by email address')
```

Comment:
Why is this being removed?

Reply:
The email shouldn't be logged out, no personal information should be logged out not even in tests really.